articles/azure-vmware/configure-external-identity-source-nsx-t.md

```diff
@@ -1,82 +1,118 @@
 ---
-title: Configure external identity source for NSX-T Data Center
-description: Learn how to use the Azure VMware Solution to configure an external identity source for NSX-T Data Center.
+Title: Configure external identity source for NSX-T Data Center
+Description: Learn how to use the Azure VMware Solution to configure an external identity source for NSX-T Data Center.
 ms.topic: how-to
 ms.service: azure-vmware
-ms.date: 10/17/2022
+ms.date: 10/23//2023
 
 ---
-# Configure external identity source for NSX-T Data Center
+# Configure an external identity source for NSX-T Data Center
+
+In this article, you will learn how to configure an external identity source for the NSX-T Data Center in an Azure VMware Solution. The NSX-T Data Center can be configured to use aexternal LDAP directory service to authenticate users, enabling a user to login using their Active Directory account credentials, or those from a 3rd party LDAP server. The account can be then be assigned an NSX-T Data Center Role, like you have with on-premises environments, to provide RBAC for each NSX-T user.
+
+![image](https://github.com/MicrosoftDocs/azure-docs/assets/22599225/b6ca2c27-849a-47f6-ba48-829db3c033c0)
```

Comment:
```suggestion

```
images must be hosted in the correct folder in the repo, and have the correct alt text, etc. @jjaygbay1 - you can add this image in a new PR if you need it.